EnglishFilter/sections/specific.txt

```diff
@@ -254,6 +254,8 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+prothomalo.com##div[class^="breakingNewsSlider-m__ad-bottom__1_vi5 breakingNewsSlider-m__show-ad__1wmb6"]
+prothomalo.com##div[class^="row1-m__inner_ads__3gOeM"]
```

Comment:
Exchange them to the rules below and move it to the `foreign.txt` -> `!------------------------------ Bengali ------------------------------` section.
```
! https://github.com/AdguardTeam/AdguardFilters/issues/72955
prothomalo.com##div[class^="breakingNewsSlider-m__ad-bottom"]
prothomalo.com##div[class^="row1-m__inner_ads_"]
```